mcs/class/System/System.Diagnostics/PerformanceCounterCategory.cs

```diff
@@ -283,7 +300,12 @@ public static bool InstanceExists (string instanceName, string categoryName, str
 			CheckCategory (categoryName);
 			if (machineName == null)
 				throw new ArgumentNullException ("machineName");
-			int val = InstanceExistsInternal (instanceName, categoryName, machineName);
+
+			//?FIXME: machine appears to be wrong
```

Comment:
nit: `//FIXME` so the `?` doesn't confuse grep searches

Reply:
ok